modules/ding_event/ding_event.features.inc

```diff
@@ -40,5 +40,6 @@ function ding_event_node_info() {
       'help' => t('This is the heading usually displayed with a big font size (h1) on the top of the page.'),
     ),
   );
+  drupal_alter('node_info', $items);
```

Comment:
Why is this drupal_alter added and is it related to features update?

Reply:
The new version of features add this.

Reply:
Okay